### ISSUE TITLE
Jet Booking - Be able to select specific and highlighted dates in the 'checkin-checkout' calendar

### ISSUE THREAD

**Issue by cAbellanosa** [CrocoBlock/suggestions#2502]
Hi all, I'm creating a booking site based on specific dates like events (outdoor trips to be specific). I would love to have the possibility to greyed out the hole calendar and be able to select specific dates that will appeared colored or highlighted in the calendar.

There is a plugin that matches this pretty well, you can see it here: https://youtu.be/Q7gbK5i4DPU?t=495

Due to JetBooking is actually based on hotel/rooms/houses reservations; maybe it could be the starting point for Jet Events...

Best,

Carlos Abellanosa

**Comment by Floydtm** [comment]
Hi! It seems that for your case it is better to use the jetappointments plugin with a custom schedule

**Comment by cAbellanosa** [comment]
Hi Floydtm; it seems, but it doesn't. Outdoor trips are based on day trips; and its true I can set an specific day with an unique hour (8 am i.e.). but I'm not sure I can sell 20 tickets for a single schedule. Trips dates are quite casual due to weather conditions and locations (we manage 9 spots) so it is not possible for us to set the same trip every Saturday i.e.
It's also quite ugly to see almost full month days strikethrough and a couple of days available...
I chatted with Crocoblock and I cannot neither show the tickets left in the form; they said its possible on calendar (?).
In my opinion there's a gap to cover for events like music, theater, conferences, exhibitions,... that are not based on schedule patterns or periods of time.
I found a head ache to adapt my needs from another situation.
Right now the best choice is WooEvents - Calendar and Event Booking; it matches 99% my needs; but I open this discussion to think about Jet Events ;)
Best
